modules/claims_api/lib/bgs_service/local_bgs.rb

```diff
@@ -114,6 +115,8 @@ def make_request(endpoint:, action:, body:, key: nil) # rubocop:disable Metrics/
                         })
       end
 
+      soap_error_handler.handle_errors(response)
```

Comment:
Wondering if we should add a check for a nil response here? I hacked the LocalBGS make_response method so it couldn't get a valid response from BGS (`'Content-Type' => 'text/xml;charset=UTF-8TootyFrooty'`) to force the condition, and got an undefined method error in soap_error_handler. Kind of contrived, but there may be cases where we don't get any response from BGS.

```
2023-04-25 13:30:44.714767 E [90673:puma srv tp 006 sentry_logging.rb:57] Rails -- undefined method `[]' for nil:NilClass.
2023-04-25 13:30:44.714932 E [90673:puma srv tp 006 sentry_logging.rb:57] Rails -- /Users/matthewchristianson/development/vets-api/modules/claims_api/lib/claims_api/error/soap_error_handler.rb:17:in `handle_errors'
/Users/matthewchristianson/development/vets-api/modules/claims_api/lib/bgs_service/local_bgs.rb:118:in `make_request'
```